srcpkgs/cdk/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'cdk'
+pkgname=cdk
+version=5.0
+revision=1
+_date=20190303
+wrksrc="cdk-${version}-${_date}"
+build_style=gnu-configure
+configure_args="--with-shared --with-ncurses=${XBPS_CROSS_BASE}/usr/lib"
+makedepends="ncurses-devel"
+short_desc="Curses Development Kit - widget library, enhanced version"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-2.0-only, BSD-4-Clause"
+homepage="https://invisible-island.net/cdk/"
+# Upstream's ftp causes travis to timeout, possilby buildbot
+distfiles="https://fossies.org/linux/misc/cdk-${version}-${_date}.tgz"
+checksum=adad7870988d44b95df57722ab8dffc587d035183eb6b12a9500ebed4d8dba25
+
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends="ncurses-devel"
+	short_desc+=" -development files"
+	pkg_install() {
+		vmove usr/bin/cdk5-config
+		vmove usr/include
```

Comment:
no static archive?

Reply:
It did not generate one with shared enabled

Reply:
get rid of it, we do care for static stuff... musl is our friend

Reply:
I will see if I can make it gen a static as well later